SECURITY.md

```
@@ -0,0 +1,46 @@
+
+| Version | Supported          |
+| ------- | ------------------ |
+| latest  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability within ACOR, please report it responsibly.
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them privately through [GitHub Security Advisories](https://github.com/skyoo2003/acor/security/advisories/new).
```

markdownlint-cli2 (0.22.0)

[warning] Link and image style

(MD054, link-image-style)